models/attribute-group/src/builder.spec.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
+import { createBuilderSpec } from '@commercetools-test-data/core/test-utils';
+import type { TAttributeGroup, TAttributeGroupGraphql } from './types';
+import * as AttributeGroup from './index';
+
+describe('builder', () => {
+  it(
+    ...createBuilderSpec<TAttributeGroup, TAttributeGroupGraphql>(
+      'graphql',
+      AttributeGroup.random(),
```

Comment:
The generator populates all the properties but the `attributes` one.
It would be great if we can add them in the test.

Reply:
Done in https://github.com/commercetools/test-data/pull/652/commits/ac5cfe5c389dddd194f3880105d4b470b2da6899